clients/admin-ui/src/pages/login.tsx

```diff
@@ -87,22 +81,15 @@ const useLogin = () => {
       return undefined;
     }
 
-    const configWizardFlagIsActive = Flags.some(
-      (flag) => flag.name === "configWizardFlag" && flag.isActive
-    );
-
-    if (configWizardFlagIsActive) {
-      if (!isSystemsLoading && systems && systems.length > 0) {
-        const datamapRoute = resolveLink({
-          href: DATAMAP_ROUTE,
-          basePath: "/",
-        });
+    if (!isSystemsLoading && systems && systems.length > 0) {
+      const datamapRoute = resolveLink({
+        href: DATAMAP_ROUTE,
+        basePath: "/",
+      });
 
-        return features.plus ? datamapRoute.href : SYSTEM_ROUTE;
-      }
-      return CONFIG_WIZARD_ROUTE;
+      return features.plus ? datamapRoute.href : SYSTEM_ROUTE;
     }
-    return INDEX_ROUTE;
```

Comment:
this was weird to me, that we redirect to index if the config wizard flag wasn't active. or maybe it's weird to redirect to the system route upon login as opposed to the index route. 

I think there were conflicting requirements [here](https://github.com/ethyca/fidesctl-plus/issues/230#issuecomment-1322557765) so I just chose one of them and went for it (though honestly I think redirecting to index makes more sense)